src/components/Table/GridTable.tsx

```diff
@@ -597,7 +600,7 @@ const VirtualRoot = memoizeOne<
     firstLastColumnWidth: number | undefined,
     xss: any,
   ) => Components["List"]
->((gs, columns, id, firstLastColumnWidth, xss) => {
+>(function VirtualRoot(gs, columns, id, firstLastColumnWidth, xss) {
```

Comment:
Note: I was trying to get a better name for this in the profile than `Anonymous Memo`, but it didn't work :(

Might be worth trying on the forwardRef below instead 🤔